lib/src/main/java/org/openbase/bco/registry/lib/com/AuthorizationFilter.java

```diff
@@ -0,0 +1,171 @@
+package org.openbase.bco.registry.lib.com;
```

Comment:
Move to authentication.lib

Reply:
rejected out of dep reasons

Reply:
Done after modifying the class and its usages a bit